precompile/config_test.go

```diff
@@ -0,0 +1,63 @@
+// (c) 2022 Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package precompile
+
+import (
+	"math/big"
+	"testing"
+
+	"github.com/ava-labs/subnet-evm/commontype"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/stretchr/testify/require"
+)
+
+func TestVerifyPrecompileUpgrades(t *testing.T) {
+	admins := []common.Address{{1}}
+	tests := []struct {
+		name          string
+		config        StatefulPrecompileConfig
+		expectedError string
+	}{
+		{
+			name:          "invalid allow list config in tx allowlist",
```

Comment:
These all use `NewTxAllowListConfig`. Can we update this to use each allow list based precompile as well as for the allow list config itself?

We can change the type for `config` above to just `Verify() error`

Reply:
Can we also add test cases just for the allow list config for:

- both are empty and it passes
- both are non-empty with no overlaps and they pass

Reply:
done